en/assembly/quick_start_pixhawk4.md

```diff
@@ -26,33 +30,29 @@ Pixhawk 4 should be mounted on the frame using vibration-damping foam pads (incl
 
 ## GPS + Compass + Buzzer + Safety Switch + LED
 
-Attach the provided GPS with integrated compass, safety switch, buzzer and LED to the GPS port.
+Attach the provided GPS with integrated compass, safety switch, buzzer and LED to the **GPS MODULE** port.
 
-> **Note** The diagram shows a GPS with an integrated compass.  The GPS/Compass should be mounted on the frame as far away from 
-  other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other 
-  electronics will reduce interference).
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
 
-![Connect compass/GPS to Pixhawk 4](../../images/pixhawk4_compass_gps.png)
+![Connect compass/GPS to Pixhawk 4](../../assets/flight_controller/pixhawk4/pixhawk4_compass_gps.jpg) 

```

Comment:
Image pretty low quality (just as bad before I reduced it). WOuld be good to get image with enough detail to see safety switch on the GPS module.